include/zephyr/sys/util_macro.h

```diff
@@ -93,6 +93,14 @@ extern "C" {
  */
 #define IS_BIT_MASK(m) IS_SHIFTED_BIT_MASK(m, 0)
 
+/**
+ * @brief Check if bit is set in a value
+ *
+ * @param value Value that contain checked bit
+ * @param bit Bit number
+ */
+#define IS_BIT_SET(value, bit) ((((value) >> (bit)) & (0x1)) != 0)
```

Comment:
Expressing this as a right shift seems needlessly confusing?  `((BIT(bit) & (value)) != 0)` is shorter and has fewer parens, FWIW.

Reply:
On systems with 32 bit `unsigned long` the `((BIT(bit) & (value)) != 0)` will not work properly for higher 32 bits of 64 bit value, due to shifting `1UL` for more bits than it has. Current approach doesn't have such issue.